homeassistant/components/reolink/host.py

```diff
@@ -26,6 +27,7 @@
 DEFAULT_TIMEOUT = 60
 FIRST_ONVIF_TIMEOUT = 15
 SUBSCRIPTION_RENEW_THRESHOLD = 300
+POLL_INTERVAL_NO_PUSH = 2
```

Comment:
The lowest polling usually allowed is 5s.

Reply:
Changed to 5 seconds.